Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -159,4 +167,30 @@ private static void extractTarFileToFolder(File file, File destinationFolder) {
         tarGZipUnArchiver.setSourceFile(file);
         tarGZipUnArchiver.extract("", destinationFolder);
     }
+
+    private static String fetchLatestStableChromeDriverVersion() {
+
+        String version = null;
+        try {
+
+            URL url = new URL(CHROME_DRIVER_LATEST_RELEASE_URL);
+            URLConnection urlConnection = url.openConnection();
+
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(urlConnection.getInputStream()));
+
+            StringBuilder content = new StringBuilder();
+            content.append(bufferedReader.readLine());
+
+            bufferedReader.close();
+
+            version = content.toString();
+
+            logger.info("Latest Chrome Driver Release found: " + version);
+
+        } catch(Exception exception){
```

Comment:
Is there possibility to use here some more specific exception ot it must be catch all?

Reply:
Codacy already remarked on that one and I changed it. :)

Reply:
Now just make them together in one call :D

> 'catch' branch identical to 'MalformedURLException' branch

Reply:
They could be handled in the same way since they are both IOExceptions in the end. But I kept them separate to distinguish between a wrongly URL and a failed download by changing the log message.